manifests/init.pp

```diff
@@ -9,6 +9,11 @@
 # $slaveof: IP address of the initial master Redis server 
 # $version: The package version of Redis you want to install
 # $packages: The packages needed to install redis
+# $redis_conf: The configuration file for redis
+# $redis_service: The serivce to run for redis
+# $protected - As of version 3.2 you need to set proteced-mode yes or no
```

Comment:
This module does not support setting the `bind` configuration item.  I would remove all references to setting the bind IP as an option.